vocdoni/bridge-ui
# Feat: Husky implemented

# Short description
Since we are going to start working as a team, we should add some code formatter so when a new PR opens the code style is consistent

# How can it be tested
- Checkout to this branch,
- Open any file and modify the style 
- Run `yarn lint:check`, it will format the changed files
- Check the review app and make sure everything is working as before :-D
# Tracker ID
N/A
